recipes.js

```diff
@@ -1,11 +1,83 @@
 const mongoose = require('mongoose');
 const Schema   = mongoose.Schema;
 const data = require('./data.js')
+var currentDate = new Date();
+
+const recipeSchema = new Schema({
+  title: {
+    type: String,
+    required: true, 
+    unique: true,
+  },
+  level: {
+    type: String, 
+    enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"],
+  },
+  ingredients: [String],
```

Comment:
Array is a type as well, instead of putting the type String inside brackets, you can write it like that : 
```
ingredients: Array,
```